www/styles/admin-swat-local.css

```diff
@@ -22,6 +20,54 @@
 
 .swat-frame {
 	background: #fff;
+	border: 0;
+	border-radius: 2px;
+	box-shadow: 0 0 2px rgba(0,0,0,0.12), 0 2px 2px rgba(0,0,0,0.24);
+}
+
+.swat-frame .swat-frame-title {
+	background: hsl(207, 72%, 53%);
+	color: #fff;
+	border-radius: 2px 2px 0 0;
+	box-shadow: 0 1px 2px rgba(0,0,0,0.24);
```

Comment:
The box shadow on the parent frame is a lighter shadow because it is the shadow on a white element. For the header, the shadow is made darker because the header is darker. This ends up making shadows that look relatively consistent instead of looking like the shadow disappears at the header.